HomeWork 5/LoggerAplication/LoggerAplication/Logger.cs

```diff
@@ -0,0 +1,50 @@
+﻿using System.Text;
+
+namespace LoggerAplication
+{
+    internal class Logger
+    {
+        private DateTime _logTime;
```

Comment:
We should use DateTime.UtcNow; otherwise, time of the error and log time can be different

Reply:
hhhhmmm.... i don`t know about that))))